### ISSUE TITLE
Update fails if symbolic link used for output folder

### ISSUE THREAD

**Issue by schoenid** [comfyanonymous/ComfyUI#6224]
### Expected Behavior

The update should ignore the output location, if a symbolic link is set instead.
![image](https://github.com/user-attachments/assets/8be62a37-e7a4-4345-866d-97e46e6d9223)

### Actual Behavior

The update stops, if a symbolic link is made instead of the output folder. There is no message in the Manager about the failed update, but the message "ComfyUI has been sucessfully updated", which is not true, because the update has been aborted.
![image](https://github.com/user-attachments/assets/ca69eb49-78e0-45c4-89d1-e39cafdaa49b)

### Steps to Reproduce

1. Remove the output folder in an installation and replace it trough a symbolic link, which points to another location for storing pictures in a more cofortable place.
2. Try to update ComfyUI.

### Debug Logs

```powershell
No logs available. I found no message about the problem.
```
Version Information from the console after an update today: 
`### ComfyUI Version: v0.2.2 | Released on '2024-09-05'`

Version Information after the update has been made AND after removing the symbolic link:
`### ComfyUI Version: v0.3.9-26-g19a64d62 | Released on '2024-12-25'`

**Comment by ltdrdata** [comment]
ComfyUI updates are supported through git functionality. Changing the path within the repo to a symbolic link is considered a modification by git, which prevents proper updates.  

If you want to use an external path for the output, use the `--output-directory` option instead of a symbolic link.

**Comment by schoenid** [comment]
Thank you for the hint.
This works fine.

I used the HowTo on https://comfyui-wiki.com/en/faq/how-to-change-output-folder. There's only a little hint, that states, using a symbolic link could affect ComfyUI updates under "**Important notes**". I didn't see it the first time. Maybe it would be more helpful to write this directly in "**Method 3: Using Symbolic Links (Advanced Users)**"?

I'm using stability matrix, so I don't have any bat file, where I could enter the options. So I took the symbolic link method.
Now I found, that I can specify the `--output-directory` option in the Launch Options under "Extra Launch Arguments".

Btw: is there a list of all the launch options of ComfyUI? (I did not find any.)
